service/src/main/java/uk/gov/hmcts/reform/fpl/service/ManageHearingsService.java

```diff
@@ -315,36 +316,15 @@ public UUID getSelectedHearingId(CaseData caseData) {
 
     public Set<String> caseFieldsToBeRemoved() {
         return Set.of(
-            "hearingType",
-            "hearingTypeDetails",
-            "hearingVenue",
-            "hearingVenueCustom",
-            HEARING_START_DATE,
-            HEARING_END_DATE,
-            "sendNoticeOfHearing",
-            "judgeAndLegalAdvisor",
-            "noticeOfHearingNotes",
-            "previousHearingVenue",
-            "firstHearingFlag",
-            "adjournmentReason",
-            "vacatedReason",
-            HEARING_DATE_LIST,
-            PAST_HEARING_LIST,
-            FUTURE_HEARING_LIST,
-            HAS_HEARINGS_TO_ADJOURN,
-            HAS_HEARINGS_TO_VACATE,
-            HAS_EXISTING_HEARINGS_FLAG,
-            HAS_FUTURE_HEARING_FLAG,
-            "hearingReListOption",
-            HEARING_START_DATE_LABEL,
-            "showConfirmPastHearingDatesPage",
-            HEARING_END_DATE_LABEL,
-            "confirmHearingDate",
-            "hearingStartDateConfirmation",
-            "hearingEndDateConfirmation",
-            START_DATE_FLAG,
-            END_DATE_FLAG,
-            "hasSession");
+            "hearingType", "hearingTypeDetails", "hearingVenue", "hearingVenueCustom", HEARING_START_DATE,
```

Comment:
Minor: I'd keep them line by line to spot diffs easily during reviews, but fine like that also.